src/HubSpot/LineItems/IHubSpotLineItemConnector.cs

```diff
@@ -7,7 +8,7 @@ namespace HubSpot.LineItems
 {
     public interface IHubSpotLineItemConnector
     {
-        Task<TLineItem> GetAsync<TLineItem>(ILineItemSelector selector) where TLineItem : LineItem, new();
-        Task<TLineItem> GetBySKUAsync<TLineItem>(ILineItemSelector selector, string sku) where TLineItem : LineItem, new();
+        Task<TLineItem> GetAsync<TLineItem>(ILineItemSelector selector, Property[] properties) where TLineItem : LineItem, new();
+        Task<TLineItem> GetBySKUAsync<TLineItem>(ILineItemSelector selector, string sku, Property[] properties) where TLineItem : LineItem, new();
     }
```

Comment:
With this change, do we need to specify all the properties when calling the sdk? Or just the "custom" ones?

Reply:
Whichever property we need data, so no matter if it's made by keystone or hubspot.

Reply:
Hmm, okay, we can't make it so that the hubspot "defaults" are always included, and then we add the extra "custom" properties through the sdk call?

Reply:
Yes it is possible, it's just something that didn't cross my mind at the time. I have fixed it now.